
package damuzhi;

option java_package = "com.damuzhi.travel";
option java_outer_classname = "CityOverviewProtos";

message CommonOverview {
  repeated string images = 1;      // 图片文件名，相对路径，如 images/a.jpg，在线返回网站绝对的URL路径
  required string html = 2;        // 对应的HTML文件名，离线返回相对路径，如 html/abc/abc.html，在线返回网站的绝对URL
}

message CityOverview {
  optional CommonOverview cityBasic = 1;               // 城市概况
  optional CommonOverview travelPrepration = 2;        // 旅行准备
  optional CommonOverview travelUtility = 3;           // 实用信息
  optional CommonOverview travelTransportation = 4;    // 城市交通
}
